.github/workflows/docs-link-cheker.yml

```
@@ -0,0 +1,41 @@
+name: Docs Links Checker
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
```

I think we don't want to run it on every push/pull-request, but once per day should suffice?

```suggestion
on:
  schedule:
    - cron: "0 9 * * *" # run daily at 9:00 UTC
```

---

That's also how I understood @antonymilne initial instruction, but please double-check with him

> We could split the link-checking job off the docs build into something that runs daily. We'd need some way of noticing when it fails, e.g. send us an alert or raise a PR (which couldn't fix anything without manual intervention but would be a good way of flagging the problem).